src/hyperlight_host/benches/benchmarks.rs

```diff
@@ -14,6 +14,14 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
+// Benchmarks are only meaningful and should only run with release builds.
+// Debug builds have different performance characteristics and would not provide
+// useful benchmarking data for performance regression testing.
+#[cfg(debug_assertions)]
+compile_error!(
```

Comment:
Sometimes I run benchmarks in release mode with debug symbols, for better stacktraces. Can we check opt level instead of debug symbols?

Reply:
@ludfjig do you feel good with the changes made here?